src/ODataHttpClient/ODataHttpClient.csproj

```diff
@@ -8,8 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNet.WebApi.Client" Version="[5.2.4,)" />
-    <PackageReference Include="Newtonsoft.Json" Version="[10.0.1,)" />
```

Comment:
Newtonsoft.Json is the backbone of this library.
Be sure not to delete it so that you can specify the version.

Reply:
OK, fixed https://github.com/iwate/ODataHttpClient/pull/16/commits/e424860ee3bd8b96a3b03d59412d8e916665ff7e